lua/modules/item_editor.lua

```diff
@@ -688,111 +688,115 @@ end
 
 Pointer.Click = function() end
 click = function(e)
-	if currentMode == mode.edit then
-		local impact
-		local shape
-		local impactDistance = 1000000000
-		for _, subShape in ipairs(shapes) do
-			if subShape.IsHidden == false then
-				local tmpImpact = e:CastRay(subShape)
-				-- if tmpImpact then print("HIT subShape, distance =", tmpImpact.Distance) end
-				if tmpImpact and tmpImpact.Distance < impactDistance then
-					shape = subShape
-					impactDistance = tmpImpact.Distance
-					impact = tmpImpact
-				end
+	if currentMode ~= mode.edit then
+		return
+	end
+	local impact
+	local shape
+	local impactDistance = 1000000000
+	for _, subShape in ipairs(shapes) do
+		if subShape.IsHidden == false then
+			local tmpImpact = e:CastRay(subShape)
+			-- if tmpImpact then print("HIT subShape, distance =", tmpImpact.Distance) end
+			if tmpImpact and tmpImpact.Distance < impactDistance then
+				shape = subShape
+				impactDistance = tmpImpact.Distance
+				impact = tmpImpact
 			end
 		end
-		if not continuousEdition then
-			if currentEditSubmode == editSubmode.pick then
-				if Player.IsHidden == false then
-					for _, bodyPartName in ipairs(bodyParts) do
-						local bodyPart = Player[bodyPartName]
-						if bodyPart.IsHidden == false then
-							local tmpImpact = e:CastRay(bodyPart)
-							-- if tmpImpact then print("HIT bodyPart, distance =", tmpImpact.Distance) end
-							if tmpImpact and tmpImpact.Distance < impactDistance then
-								impactDistance = tmpImpact.Distance
-								impact = tmpImpact
-							end
-						end
+	end
+
+	if continuousEdition then
+		return
+	end
```

Comment:
Note: in this case we don't do
```lua
	if impact ~= nil then
		checkAutoSave()
		refreshUndoRedoButtons()
	end
```
It shouldn't happen and no block were added so it's fine for me